qa/L0_unit_tests/test.sh

```diff
@@ -25,7 +25,7 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 TEST_LOG='test.log'
-EXPECTED_NUM_TESTS=19
+EXPECTED_NUM_TESTS=20
```

Comment:
Do you need to increase this number for every new unit test? That is not a good idea or check.

Reply:
I can remove it altogether if you think it would be better but currently, we need to increase it for every new test.

Reply:
This should be dynamic. I should be able to freely add new tests without changing any other code. Let's try to make that possible.

Reply:
I fixed this.